packages/ui/src/helpers/authenticator/formFields/defaults.ts

```diff
@@ -158,6 +157,7 @@ const getForceNewPasswordFormFields = (state: AuthMachineState): FormFields => {
       formField[fieldName] = { ...getDefaultFormField(fieldName) };
     } else {
       // There's a `custom:*` attribute or one we don't already have an implementation for
+      // eslint-disable-next-line no-console
```

Comment:
Linting fix due to new rules ([ref](https://github.com/aws-amplify/amplify-ui/pull/5830/files#diff-6e97cd3294540e00be8979d5d20d4f092e2d49d3d7e796d472f3e03961d69ab5R31))